documentation/asciidoc/computers/raspberry-pi/frequency-management.adoc

```diff
@@ -2,8 +2,6 @@
 
 All Raspberry Pi models perform a degree of thermal management to avoid overheating under heavy load. The SoCs have an internal temperature sensor, which software on the GPU polls to ensure that temperatures do not exceed a predefined limit; this is 85°C on all models. It is possible to set this to a lower value, but not to a higher one. As the device approaches the limit, various frequencies and sometimes voltages used on the chip (ARM, GPU) are reduced. This reduces the amount of heat generated, keeping the temperature under control.
 
-When the core temperature is between 80°C and 85°C, a warning icon showing a red half-filled thermometer will be displayed, and the ARM cores will be progressively throttled back. If the temperature reaches 85°C, an icon showing a fully filled thermometer will be displayed, and both the ARM cores and the GPU will be throttled back. See the page on xref:configuration.adoc#firmware-warning-icons[warning icons] for images of the icons.
```

Comment:
The bit you've removed about the ARM cores and/or GPU being throttled back **is** still relevant, even if the associated warning-icon is no longer displayed.

Reply:
Oops. Yes. That needs to get put back. The only thing that should have been removed was " See the page on xref:configuration.adoc#firmware-warning-icons[warning icons] for images of the icons." Can you raise a PR to put this back @jehlers42. Thanks.

Reply:
Fixed by #3069`